doc/sphinx/introduction/how_to_use_this_manual.md

```diff
@@ -6,40 +6,40 @@ The manual is a large document with a lot of information for different audiences
 
 5. **Parameter Listings**
 
-   This part is important for all users. It contains a list of all the parameters which are available in the world builder input files and the GWB tools.
+   This part is important for all users. It contains a list of all available parameters in the world builder input files and the GWB tools.
 
 6. **Project Links**
 
-   This part contains links which are relevant to the project.
+   This part contains links relevant to the project.
 
 ## Using the Functionality
 
 Now that you know what to find where in the manual, let's explain the interface and how to use it.
 
-**Left panel** We already talked about the left panel in the previous section, but there are a few other things to note. First of all, there is a search box, which may help you to find the content you need. Secondly, when you click on the logo, you are taken back to the opening page of the manual. Lastly, you can hide the side panel, by clicking the icon with the three horizontal bars at the top, just right from the left side panel. 
+**Left panel** We already talked about the left panel in the previous section, but there are a few other things to note. First of all, there is a search box, which may help you to find the content you need. Secondly, when you click on the logo, you are taken back to the opening page of the manual. Lastly, you can hide the side panel by clicking the icon with the three horizontal bars at the top, just right from the left side panel. 
 
 **Top panel** This top panel contains on the left side the button to hide and reveal the left panel, and on the right side three very useful buttons. The left most of these buttons sets the page in full screen. Unfortunately, it will currently not stay in full screen when switching between pages. The middle button is probably the most important one. It opens a drop down menu with a direct link to the GWB GitHub repository, a direct link to open a new issue and a direct link to edit the page you are currently on.  So if you see a mistake or something is missing on the page, it is very easy to fix without needing to download anything! The rightmost button allows you to download the page you are currently on.
 
 **Right panel** The right panel contains an outline of the page you are on. This is especially useful for longer pages.
 
-**Navigation** At the bottom of each page there are buttons to go the the next and previous page. You can also use the arrow keys instead to move the the next and previous page.
\ No newline at end of file
+**Navigation** At the bottom of each page there are buttons to go to the the next and previous page. You can also use the arrow keys instead to move to the the next or previous page.
```

Comment:
Can you also remove the two  `the the` on this line?

Reply:
Fixed it (and also found a third spot in the same file).